src/main/java/frc/robot/Constants.java

```diff
@@ -13,18 +15,32 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
-  }
-
-    public static final CANAddress SWERVE_TURN_LEFT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_TURN_LEFT_REAR = new CANAddress(0);
-    public static final CANAddress SWERVE_TURN_RIGHT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_TURN_RIGHT_REAR = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_LEFT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_LEFT_REAR = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_RIGHT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_RIGHT_REAR = new CANAddress(0);
+    public static class OperatorConstants {
+        public static final int kDriverControllerPort = 0;
+    }
+
+    public static final CANAddress SWERVE_TURN_LEFT_FRONT = new CANAddress(1);
+    public static final CANAddress SWERVE_DRIVE_LEFT_FRONT = new CANAddress(2);
+    public static final CANAddress SWERVE_MODULE_ENCODER_LEFT_FRONT = new CANAddress(9);
+
+    public static final CANAddress SWERVE_TURN_RIGHT_FRONT = new CANAddress(3);
+    public static final CANAddress SWERVE_DRIVE_RIGHT_FRONT = new CANAddress(4);
+    public static final CANAddress SWERVE_MODULE_ENCODER_RIGHT_FRONT = new CANAddress(10);
+
+    public static final CANAddress SWERVE_DRIVE_LEFT_REAR = new CANAddress(5);
+    public static final CANAddress SWERVE_TURN_LEFT_REAR = new CANAddress(6);
+    public static final CANAddress SWERVE_MODULE_ENCODER_LEFT_REAR = new CANAddress(11);
+
+    public static final CANAddress SWERVE_DRIVE_RIGHT_REAR = new CANAddress(7);
+    public static final CANAddress SWERVE_TURN_RIGHT_REAR = new CANAddress(8);
+    public static final CANAddress SWERVE_MODULE_ENCODER_RIGHT_REAR = new CANAddress(12);
```

Comment:
This is all redundant. We've already informed YAGSL about the CAN IDs in its configuration files (the `.json` files), so we don't need to duplicate the values here. Having them here creates more potential for confusion or mismatches.